### PR TITLE
Combine ControlNet scripts to one script

### PR DESCRIPTION
**Pull request by dilithjay**
This PR combines the ControlNet scripts into one function that accepts the ControlNet mode as a parameter.

It also adds support for the rest of the ControlNet modes.

**Using guess mode:**
Screen Shot 2023-05-21 at 6 35 52 PM

**Without guess mode, manual adjustment:**
Screen Shot 2023-05-21 at 6 30 42 PM

Screen Shot 2023-06-10 at 10 20 50 PM

#24

### CODE REVIEW

File: img_styler/image_prompt/control_net/scribble2image.py
Comment (pramitchoudhary) [inline img_styler/image_prompt/control_net/scribble2image.py:1]: Is there any need to keep `canny2image.py` and `scribble2image.py`?

Comment (dilithjay) [reply img_styler/image_prompt/control_net/scribble2image.py:1]: They are deleted, right?

Comment (pramitchoudhary) [reply img_styler/image_prompt/control_net/scribble2image.py:1]: Yes, my bad.

**Review by pramitchoudhary — APPROVED** [review]
Thanks @dilithjay 🙏

**Pull request merged**